.gitignore

```diff
@@ -1,5 +1,5 @@
 node_modules/
-./solidity/truffle.js
+./solidity-v1/truffle.js
```

Comment:
This file no longer exists, could be a good opportunity to clean up.

Reply:
0d674f7